Server/src/cli/utils/output.py

```
@@ -0,0 +1,195 @@
+    """Print an error message to stderr."""
+    click.echo(f"❌ {message}", err=True)
+
+
+def print_warning(message: str) -> None:
+    """Print a warning message."""
+    click.echo(f"⚠️  {message}")
+
+
+def print_info(message: str) -> None:
+    """Print an info message."""
+    click.echo(f"ℹ️  {message}")
```

Ruff (0.14.13)

String contains ambiguous `ℹ` (INFORMATION SOURCE). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)